src/masternodes/govvariables/attributes.h

```diff
@@ -145,16 +153,19 @@ struct CDexTokenInfo {
 };
 
 using CDexBalances = std::map<DCT_ID, CDexTokenInfo>;
+using OracleSplits = std::map<uint32_t, int32_t>;
+using DescendantValue = std::pair<uint32_t, int32_t>;
+using AscendantValue = std::pair<uint32_t, std::string>;
 using CAttributeType = std::variant<CDataStructureV0>;
-using CAttributeValue = std::variant<bool, CAmount, CBalances, CTokenPayback, CDexBalances, CTokenCurrencyPair>;
+using CAttributeValue = std::variant<bool, CAmount, CBalances, CTokenPayback, CTokenCurrencyPair, OracleSplits, DescendantValue, AscendantValue, CDexBalances>;
```

Comment:
We could add comment that ordering is important, deserialized type is decide via the index. So it's better `CDexBalances` to stay after `CTokenPayback` just in case someone is running the master branch (it's not in official release but there is a PR)

Reply:
Sure, once everything is merged into v3 lets add a comment about the ordering.